src/main/java/com/google/cloud/tools/gradle/appengine/core/DownloadCloudSdkTask.java

```diff
@@ -66,6 +66,6 @@ public void downloadCloudSdkAction() {
       }
     }
 
-    cloudSdkBuilderFactory.setCloudSdkHome(sdkHome);
+    this.cloudSdkBuilderFactory.setCloudSdkHome(sdkHome);
```

Comment:
I believe `this` is not necessary if `cloudSdkBuilderFactory` is not also a local variable.